audiotools/core/ffmpeg.py

```diff
@@ -61,20 +62,26 @@ def r128stats(filepath: str, quiet: bool):
     return stats_dict
 
 
-def ffprobe_offset(path):
+def ffprobe_offset_and_codec(path: str) -> Tuple[float, str]:
```

Comment:
wondering if we should add this new function `ffprobe_offset_and_codec` as separate from the existing `ffprobe_offset`, instead of replacing/reusing. This will help maintain backward compatibility for other places (other workers, script outside of this repo) where this function might be getting used.

Reply:
Nevermind, I serched across our org to see if `ffprobe_offset` is being used anywhere else, and didn't find any its use anywhere else. So its fine to replace it.